### ISSUE TITLE
Output of AU detector

### ISSUE THREAD

**Issue by Dongzhikang** [tomas-gajarsky/facetorch#53]
Hi Gajarsky. I really appreciate that you deveop this cool tool! But I am a little bit confused about the output of AU detector like this.

>  'au': Prediction(label='inner_brow_raiser', logits=tensor([5.9231e-01, 6.0695e-02, 3.0883e-01, 3.8449e-02, 2.9211e-01, 3.2139e-01,
        7.0077e-04, 3.4345e-01, 1.0299e-03, 8.1870e-02, 0.0000e+00, 1.3947e-01,
        3.1471e-02, 1.4108e-03, 8.3429e-02, 0.0000e+00, 6.6746e-05, 0.0000e+00,
        0.0000e+00, 1.1023e-01, 2.9374e-03, 1.3591e-01, 7.4503e-03, 3.5197e-04,
        2.9681e-03, 0.0000e+00, 0.0000e+00, 1.6567e-02, 5.1174e-03, 3.3988e-02,
        5.1338e-04, 7.6977e-04, 3.8522e-03, 4.3307e-03, 7.2081e-04, 0.0000e+00,
        2.0911e-02, 1.7393e-03, 6.8589e-03, 3.1934e-02, 0.0000e+00]), other={'multi': ['inner_brow_raiser']})

From #46 I know that the number is the logits of each AU. However, I am confused that how to interpret these logits? Does it mean the confidence of prediction of each unit? Can I know the position of each predicted unit? Or draw a nice heatmap like the original paper? Thanks!

**Comment by tomas-gajarsky** [comment]
Thank you for your kind words and for trying out facetorch. Let me help clarify the output structure of the AU detector for you.

1. Logits: The values you see under logits are indeed the confidence scores for each Action Unit (AU). Higher values represent higher confidence levels.
2. Label: The label attribute contains the name of a single AU that has the highest confidence among all AUs detected.
3. other['multi']: This captures all the AUs whose confidence scores (logits) surpass a set threshold, which is currently 0.5. If multiple AUs exceed this threshold, they will all be listed here.

Regarding your other questions:

Currently, facetorch does not have a functionality that enables you to visualize the exact position of each predicted unit.
Additionally, while the original paper might have presented heatmaps, Facetorch does not offer heatmap generation out-of-the-box.
Lastly, the exported TorchScript model used in facetorch does not output the latent node features, which could potentially be used to generate such heatmaps.
